### ISSUE TITLE
DS/AGB game doesn't boot

### ISSUE THREAD

**Issue by JustForNothing** [OperationNT414C/CtrBootManager#6]
Hi,

There is a problem with the (A9LH) ctrbootmanager, DS game doesn't boot, and other VC apps. (For Luma and Gateway)
Here: https://github.com/AuroraWright/Luma3DS/wiki/FAQ-and-Troubleshooting#gbads-gamesdsiwaresmash-brosmonster-hunter-4-fail-to-start

It work with Luma3ds (VC Game)
But not with ctrbootmanager9 :/

**Comment by OperationNT414C** [comment]
Don't have too much time today to give precise answer: in the wiki, search for "binary patches" detail, there is something specific described for Luma3DS use with VC games.

**Comment by JustForNothing** [comment]
Tried Luma3ds as arm9loaderhax.bin and VC game work.
Tried the lastest arm9loaderhax.bin from ctrbootmanager9, and VC game doesn't work with Luma3ds :/

**Comment by OperationNT414C** [comment]
Sorry, I could not give a precise answer yesturday (I was at my wedding :-) ).

The behavior you get is normal: when you start VC games (particularly GBA and DS), the 3DS restarts on the SysNAND and Luma3DS needs to "find itself" on reboot. By default, Luma3DS has an hardcoded path on "/arm9loaderhax.bin", it means that if "/arm9loaderhax.bin" isn't Luma3DS, the 3DS crashes instead of booting GBA or DS game.

There are 2 wayes to solve this issue:
- Luma3DS path changer which will modify Luma3DS ".bin" to put the path you wish instead of "/arm9loaderhax.bin"
- CtrBootManager9 binary patch feature which will dynamically edit the path on Luma3DS first load

The binary patch feature is explained in detail on CtrBootMangager wiki (see the end of the page):
https://github.com/OperationNT414C/CtrBootManager/wiki/CtrBootManager-setup

To sum up, you must put those line into your Luma3DS entry:

```
[entry]
title=Luma3DS;
path=/MyLuma3DSPath.bin;
offset=0;
patchMemSearch=730064006D0063003A002F00;
patchMemOverwriteWStr=sdmc:/MyLuma3DSPath.bin;
patchOccurence=1;
key=-1;
```

Please replace all "/MyLuma3DSPath.bin" with your Luma3DS real path. There is also a limitation at 38 characters in Luma3DS (otherwise, the binary patch feature will start to overwrite crutial Luma3DS binary code).

**Comment by JustForNothing** [comment]
Ok nice it work :)
Thanks
